pkg/ttnpb/application_interfaces.go

```diff
@@ -144,6 +144,22 @@ func (m *GetApplicationCollaboratorRequest) ExtractRequestFields(dst map[string]
 	m.GetApplicationIds().ExtractRequestFields(dst)
 }
 
+func (up *ApplicationUp) ExtractRequestFields(dst map[string]interface{}) {
```

Comment:
I'm pretty sure that in `pkg/ttnpb` we alsways use `m` as receiver. Also for the other ones below.